src/android/com/outsystems/firebase/analytics/model/OSFANLAnalyticsEvent.kt

```diff
@@ -0,0 +1,23 @@
+package com.outsystems.firebase.analytics.model
+
+enum class OSFANLAnalyticsEvent(val value: String) {
```

Comment:
Do we need this? Doesn't `FirebaseAnalytics` provide the same structure? (https://firebase.google.com/docs/reference/kotlin/com/google/firebase/analytics/FirebaseAnalytics.Event)

Reply:
Indeed. Removed.